### PR TITLE
Turn on integration tests for Jörmungandr (Part I: Addresses & Wallets scenarios)

### PR DESCRIPTION
**Pull request by paweljakubas**
# Issue Number

#358 

# Overview

- [x] I have turn on wallet/addresses scenario integration tests
- [x] I have added Buildable instance for Tx in shared
- [x] I have added feePolicy arg to newWalletLayer and enabled correct fee calculation for cardanoPolicy and jormungandrPolicy

# Comments

### CODE REVIEW

**Comment by Anviking** [comment]
I did this `git rebase -i` and pushed
```
pick 3317355a turn on addresses and wallets scenarios
pick 806b8dfe Add support for posting txs in JormungandrLayer and NetworkLayer
drop f66e6c23 logs
pick a1f24cf7 enable logging in integration tests + delay between calls in expectEventually
pick a2ba77f7 Improve on generation with retrieve and new generate option, also printing public key
drop 7463e20a Fix: mkStdTx should sign with the /current/ txId
pick 14bd8dec add buildable instance for tx
pick 359c42b1 add feePolicy arg to newWalletLayer
drop 556a24a8 remove unneeded traces

pick df81b5c7 hlint and nix
# would preferably squash hlint part but can't find where it should be 🤷‍♂ 

```

File: lib/core/src/Cardano/Wallet/Primitive/Fee.hs [hunk 1]
```diff
@@ -82,6 +83,10 @@ data TxSizeLinear =
 cardanoPolicy :: TxSizeLinear
 cardanoPolicy = TxSizeLinear (Quantity 155381) (Quantity 43.946)
 
+-- | Hard-coded zero fee policy for Jormungandr
+jormungandrPolicy :: TxSizeLinear
```
Comment (Anviking) [inline lib/core/src/Cardano/Wallet/Primitive/Fee.hs:87]: 👍 

I am a _tiny_ bit worried that our test setup will differ from `exe` setup _eventually_, but that's not just about the fee policy

**Review by Anviking — APPROVED** [review]
lgtm!

File: lib/core/src/Cardano/Wallet/Primitive/Fee.hs (hunk 1 above)
Comment (KtorZ) [reply lib/core/src/Cardano/Wallet/Primitive/Fee.hs:87]: Yeah, ideally, this should come from the genesis block for Jörmungandr. I'll make a PR A.S.A.P

**Review by KtorZ — APPROVED** [review]

**Pull request merged**